modules/framework/galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/spi/rbac/RBACService.java

```diff
@@ -17,6 +17,36 @@ public interface RBACService {
      */
     public static final String ENV_VARIABLE_GALASA_DEFAULT_USER_ROLE_NAME = "GALASA_DEFAULT_USER_ROLE";
 
+    /**
+     * This is the name of the environment variable which holds a list of user login ids, 
+     * separated by commas. Each login id is synthetically treated as being an 'owner' of the
+     * Galasa service.
+     * 
+     * The actual 'owner' role isn't set into their user record, but is just made to appear as
+     * the owner role.
+     * 
+     * Users with the owner role :
+     * - cannot have their user record deleted
+     * - cannot have their user role changed
+     * 
```

Comment:
Maybe worth mentioning here "apart from by `admin`"?

Reply:
And the service would need to be restarted?

Reply:
I say to update the Helm values and upgrade the system. That will do a rolling restart on all the services i think.... which re-starts the pod.